client/my-sites/controller.js

```diff
@@ -401,6 +408,25 @@ export function siteSelection( context, next ) {
 			return window.location.replace( pluginLink );
 		}
 
+		if (
+			has( window, 'location.replace' ) &&
+			calypsoifyGutenberg &&
+			/^\/gutenberg\//.test( basePath )
```

Comment:
```suggestion
```

Should this be testing for `gutenberg` in base path? Aren't we supposed to to redirect regular routes like `/post/{ site }`? (or am I missing some intermediate level of redirection that will point that to `/gutenberg/` first?

Reply:
There's been a discussion on Slack about this.
The idea is to keep both Calypso Classic and Gutenberg editors around.
So `/post` will continue loading Calypso Classic, and `/gutenberg/post` will load Gutenberg.
This means that for as long as we need Calypsoify, only `/gutenberg/post` will redirect.

Reply:
Please see https://github.com/Automattic/wp-calypso/pull/27317#issuecomment-435003221 because it answers this comment too.